src/UI/RenderConfig.elm

```diff
@@ -57,22 +95,49 @@ updateWindow window (RenderConfig oldData) =
         |> RenderConfig
 
 
+{-| `True` when the browser is a mobile browser.
+
+    if RenderConfig.isMobile renderConfig then
+        -- Mobile view
+    else
+        -- Desktop view
+
+-}
 isMobile : RenderConfig -> Bool
 isMobile (RenderConfig { deviceClass }) =
     deviceClass == Element.Phone
 
 
+{-| `True` when the browser is in portrait size and not landscape.
+
+    if RenderConfig.isMobile renderConfig then
+        -- Mobile view
+    else
+        -- Desktop view
+
+-}
 isPortrait : RenderConfig -> Bool
 isPortrait (RenderConfig { deviceOrientation }) =
     deviceOrientation == Element.Portrait
 
 
+{-| `RenderConfig.elLayoutAttributes` are the recommended attributes for [`Element.layout`](/packages/mdgriffith/elm-ui/latest/Element#layout).
+
+If the app is using [`UI.NavigationContainer.navigator`](UI-NavigationContainer#navigator) then there is nothing to worry about as it already uses these attributes.
+
+    Element.layout (RenderConfig.elLayoutAttributes renderConfig) (appView model)
+
+-}
 elLayoutAttributes : RenderConfig -> List (Attribute msg)
 elLayoutAttributes _ =
     -- Why here? Accessibility settings may change fonts, backgrounds, etc...
     [ Font.family [ Font.typeface "Inter", Font.sansSerif ] ]
 
 
+
+-- Internal
+
+
```

Comment:
Maybe we don't need that 🤔  I'm not sure we are using this as a pattern in all other modules

Reply:
In a few, it doesn't interfere in docs.

Reply:
Yeah, but let's try to be consistent then, add this everywhere